main.go

```diff
@@ -200,15 +201,24 @@ func main() {
 	log.Info().Msg("Shutting down...")
 }
 
-func applyJitter(input int) (output int) {
+func processHorizontalPodAutoscaler(kubeClient *k8s.Client, hpa *autoscalingv1.HorizontalPodAutoscaler, initiator string) (status string, err error) {
+	status = "failed"
 
-	deviation := int(0.25 * float64(input))
+	if &hpa != nil && &hpa.Metadata != nil && &hpa.Metadata.Annotations != nil {
```

Comment:
isn't the `&` making these variables a pointer of a pointer?

Reply:
This was already there (on [line 366 in `master`](https://github.com/estafette/estafette-k8s-hpa-scaler/blob/master/main.go#L366)), it was just moved with the reordering of the functions.

But looking at it now, I think you're right.

It should be
```
	if hpa != nil && hpa.Metadata != nil && hpa.Metadata.Annotations != nil {
```
right?